### ISSUE TITLE
Script completely bricked my machine

### ISSUE THREAD

**Issue by HadockKali** [semazurek/ET-All-in-One-Optimizer#13]
I was using the script to debloat a VMWare virtual machine when mid process both the networks on the host machine as well as the one on the VM disappeared without the possibility of restoration, system restore did nothing, and my Wi-Fi board is not visible in device manager on neither machine, the host machine is an Windows 11 laptop, what is this joke?!?!? Please advise asap.

**Comment by semazurek** [comment]
Describe exactly what's bricked. Script was tested on VirtualBox -> Windows 10 21h2 and Windows 11 as well in 20h2 + 21h2 Windows 11 and 10 on LOCAL Laptops from Dell, Lenovo, Acer, Asus and nothing like this has ever happened to anyone before.

The script does not interfere with the network settings, only there is an option to disable HotSpot sharing and access (Which does not disable the sharing settings from the HotSpot laptop).

If this is on a virtual machine, maybe the machine setup is wrong. I want to help you but don't know what's happend

**Comment by semazurek** [comment]
If there is not WiFi card on your PC/Laptop visible i suggest downloading DRIVER typing model of motherboard/network card (in case of PC) or model of laptop in google. This seems for using default windows drivers and by some reason system marked them as old and script maaybe big MAYBE and incredibly not common removed it by powershell comand.

**Comment by HadockKali** [comment]
Hello, sorry for the fiery comment, the machine was set up correctly indeed, I have to yet identify the issue, I'm looking through windows event viewer, very strange, I managed to repair it by updating the host BIOS and clean installing windows, drivers did not work, and in device manager, the Wi-Fi board was not even visible. I booted from a USB using PCMagic, and there the component was present (this does not explain why drivers didn't fix the issue). Also, MyASUS help module in windows recovery (ROG Strix G17 2022 Laptop AMD chipset). What you are proposing, the shell command might have removed the driver, saw something related during the execution of that module, and short after the connections on the host and VM dropped *poof*. If you would like me to provide logs or other technical data please don't hesitate to reach out, for now, the problem it's solved but let's get to the root cause.

**Comment by semazurek** [comment]
Cery stranger if you reinstalled fresh windows and yet invisible device maybe new windows builds have problem with it rights now...  And this could explain why PCMagic wifi worka (I guess that's Linux) 

Because everything points to a case with the fact that the network stopped working, I will think about it today after work.

**Comment by semazurek** [comment]
Well my question is still open:
1)  Did you reinstall Windows on laptop and the same problem happend ?
2) Problem is only on VM ? 
3) Your PC stuck on safe mode - becouse of antivirus blocking functions of ET Script or whatever ?

Becouse it seems from what you saying it's not couse by script but hardware problem...

**Comment by semazurek** [comment]
If your machine is "bricked" without internet connection you probably:
1) changed name of E.T. script 
2) You used remove defender or edge option and you are in safe-mode without network support.

You can easy repair this by running again E.T. script or just type command: ```bcdedit /deletevalue {current} safeboot```

**Comment by HadockKali** [comment]
Hi, thank you very much for your continued interest in this problem, I had other issues at hand and completely forgot. I did a little more digging and found out what actually might be the problem, the script was running on a Windows Enterprise VM, which I think it's not supported and might have broken something, I have remediated the problem by restoring to the backed-up image of the VM. I will try to reproduce the issue and try the above fix as well.
